build-tools/src/main/resources/software/amazon/awssdk/spotbugs-suppressions.xml

```diff
@@ -172,6 +172,14 @@
         <Bug pattern="RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE"/>
     </Match>
 
+    <!-- See SpotBugs bug: https://github.com/spotbugs/spotbugs/issues/600, https://github.com/spotbugs/spotbugs/issues/756 -->
```

Comment:
Note these bugs are fixed in SpotBugs but not yet released as of today: https://github.com/spotbugs/spotbugs/blob/0c8d6b180f175b3594b83bdd0cc6281ce7ef3e12/CHANGELOG.md#unreleased---2020--